frontend/src/locales/de-CH-scout.json

```diff
@@ -8,5 +15,12 @@
   },
   "global": {
     "language": "Deutsch (Pfadi)"
+  },
+  "views": {
+    "camp": {
+      "navigationCamp": {
+        "material": "Mat"
```

Comment:
Really needed in de-Scout?

Reply:
What do you mean by "needed"?

Reply:
The de.json value is "Mat.". Not really convinced that omitting the dot already makes it a scout slang.

Not that I have a strong opinion on that. Just a general feeling that I'd limit the Scout-specific translations to vocabulary that really is very scout specific.

Approved from my side with or without.

Reply:
Fair point, it should definitely be written out "Material" in German. I've also removed Mat from scout German for now.